README.md

```diff
@@ -9,7 +9,9 @@ DESCRIPTION TBD
 | Package | Release | Description |
 | ------- | :-----: | ----------- |
 | [account-abstraction](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/account-abstraction) | [![NPM Version](https://badge.fury.io/js/%40safe-global%2Fsafe-core.svg)](https://badge.fury.io/js/%40safe-global%2Faccount-abstraction) | TBD |
-| [relay-provider](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/relay-provider) | [![NPM Version](https://badge.fury.io/js/%40safe-global%2Frelay-provider.svg)](https://badge.fury.io/js/%40safe-global%2Frelay-provider) | TBD |
+| [auth-kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/auth-kit) | [![NPM Version](https://badge.fury.io/js/%40safe-global%2Fauth-kit.svg)](https://badge.fury.io/js/%40safe-global%2Fauth-kit) | TBD |
+| [onramp-kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit) | [![NPM Version](https://badge.fury.io/js/%40safe-global%2Fonramp-kit.svg)](https://badge.fury.io/js/%40safe-global%2Fonramp-kit) | TBD |
+| [relay-kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/relay-kit) | [![NPM Version](https://badge.fury.io/js/%40safe-global%2Frelay-kit.svg)](https://badge.fury.io/js/%40safe-global%2Frelay-kit) | TBD |
```

Comment:
Just to let you know that all the packages have "TBD" as their descriptions in case it was missed.

Reply:
Added a small description for the kits 👍